Data/Sys/GameSettings/RK5.ini

```diff
@@ -17,3 +17,5 @@ EmulationStateId = 5
 [ActionReplay]
 # Add action replay cheats here.
 
+[Video_Hacks]
+EFBToTextureEnable = True # Fix shadows
```

Comment:
Oh? If so, this is likely a fixable bug. Is there an issue report for those shadows being broken on EFB2RAM?